cloudbuild/nightly.sh

```diff
@@ -24,36 +26,19 @@ fi
 readonly DATE="$(date +%Y%m%d)"
 readonly REVISION="0.0.${DATE}"
 readonly MVN="./mvnw -B -e -s /workspace/cloudbuild/gcp-settings.xml -Dmaven.repo.local=/workspace/.repository -Drevision=${REVISION}"
-readonly BUCKET="spark-lib-daily-snapshots"
 
 cd /workspace
 
 # Build
-$MVN install -DskipTests -Pdsv1_2.12,dsv2
+$MVN install -DskipTests -Pdsv1_2.11,dsv1_2.12,dsv2
 #coverage report
-$MVN test jacoco:report jacoco:report-aggregate -Pcoverage,dsv1_2.12,dsv2
+$MVN test jacoco:report jacoco:report-aggregate -Pcoverage,dsv1_2.11,dsv1_2.12,dsv2
 # Run integration tests
-$MVN failsafe:integration-test failsafe:verify jacoco:report jacoco:report-aggregate -Pcoverage,integration,dsv1_2.12,dsv2_2.4
+$MVN failsafe:integration-test failsafe:verify jacoco:report jacoco:report-aggregate -Pcoverage,integration,dsv1_2.11,dsv1_2.12,dsv2
```

Comment:
please restore dsv2_2.4, as dsv2_3.1 integration tests are not ready

Reply:
good catch

Reply:
@suryasoma can you replace dsv2 with dsv2_2.4?